### ISSUE TITLE
Additional query params or endpoints

### ISSUE THREAD

**Issue by schliflo** [coverified/backend#3]
We need the following functionality:

- [x]  Do sorting server-side in backend
- [x]  Get latest feed entries
- [ ]  Limit returned entries per feed
- [x]  Ad endpoint to trigger the crawler via Cloud Scheduler (https://cloud.google.com/scheduler/docs/quickstart)

**Comment by johanneshiry** [comment]
I addressed the marked issues in #4 

"Limit returned entries per feed" would require major changes in the data model hence I would postpone it for now if we still know how exactly we're going to use it

**Comment by schliflo** [comment]
Sure thing. #4 looks good to me. I think we can go ahead and close this issue after #4 is merged. Then open a new one with an exact specification of what we need once we're sure what we need...
